examples/lines_bars_and_markers/linestyles.py

```diff
@@ -3,31 +3,44 @@
 Linestyles
 ==========
 
-This examples showcases different linestyles copying those of Tikz/PGF.
+This example showcases different linestyles copying those of Tikz/PGF.
+Linestyle can be provided as simple as *solid* , *dotted* or *dashed*.
+Moreover, the dashing of the line can be controlled by a dash tuple
+such as (offset, (on_off_seq)) as mentioned in `.Line2D.set_linestyle`.
+For e.g., ``(0, (3, 10, 1, 15))`` means 3pt-line,10pt-space,1pt-line,15pt-space
+with no offset.
+
+*Note*: The dash style can also be configured via `.Line2D.set_dashes`
+as shown in :doc:`/gallery/lines_bars_and_markers/line_demo_dash_control`
+and passing a list of dash sequences using the keyword *dashes* to the
+cycler in :doc:`property_cycle </tutorials/intermediate/color_cycle>`.
 """
 import numpy as np
 import matplotlib.pyplot as plt
 from collections import OrderedDict
```

Comment:
this can now be removed, since no longer an ordered dict.

Reply:
I am still using `linestyles = OrderedDict(linestyles[::-1])` here. I just reversed the list so that it follows the same order between the list description and the plot.

Reply:
Sorry about that then, but I don't think you need the ordered dict anyway?